services/jobserver.py

```diff
@@ -0,0 +1,90 @@
+import csv
+from base64 import b64encode
+from urllib.parse import urljoin
+
+from environs import Env
+
+from services import session
+
+
+env = Env()
+
+JOB_SERVER_URL = env.str("JOB_SERVER_API")
+JOB_SERVER_TOKEN = env.str("JOB_SERVER_TOKEN")
+RELEASES_URL = urljoin(
+    JOB_SERVER_URL,
+    "api/v2/releases/workspace/test-interactive",
+)
+
+
+def fetch_release(analysis_request_id):
+    headers = {"Authorization": JOB_SERVER_TOKEN}
+    response = session.get(
+        RELEASES_URL,
+        headers=headers,
+    )
+    response.raise_for_status()
+
+    output = {}
+    for file in response.json()["files"]:
+        if DecilesChart.exists(file, analysis_request_id):
+            output[DecilesChart.name] = DecilesChart.decode(file)
+        elif EventsCount.exists(file, analysis_request_id):
+            output[EventsCount.name] = EventsCount.decode(file)
+        elif CommonCodes.exists(file, analysis_request_id):  # pragma: no cover
```

Comment:
I can't work out why pytest is saying there's no coverage for this line, because there really is. Any ideas?